modules/iam-assumable-role/variables.tf

```diff
@@ -82,16 +82,10 @@ variable "custom_role_policy_arns" {
   default     = []
 }
 
-variable "custom_role_trust_policy" {
```

Comment:
this is a breaking change so we won't be making that here. a lot of these changes and nuanced differences will be resolved at the next breaking change (ref  https://github.com/clowdhaus/terraform-aws-iam)

Reply:
so do you want me to revert back the name of the variable to `custom_role_trust_policy` instead of the `assume_role_policy` ?